apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -1,282 +0,0 @@
-import {
-	CheckoutStepBody,
-	checkoutTheme,
-	CheckoutSummaryArea,
-	CheckoutSummaryCard,
-	MainContentWrapper,
-	CheckoutStepAreaWrapper,
-	SubmitButtonWrapper,
-	FormStatus,
-} from '@automattic/composite-checkout';
-import { useLocale, useLocalizeUrl } from '@automattic/i18n-utils';
-import { useSiteDomains, useDomainSuggestion, useDomainSearch, useTitle } from '@automattic/launch';
-import { Title, SubTitle, ActionButtons, BackButton } from '@automattic/onboarding';
-import { ThemeProvider } from '@emotion/react';
-import { Button, Tip } from '@wordpress/components';
-import { useDispatch, useSelect } from '@wordpress/data';
-import { createInterpolateElement } from '@wordpress/element';
-import { sprintf } from '@wordpress/i18n';
-import { Icon, check } from '@wordpress/icons';
-import { useI18n } from '@wordpress/react-i18n';
```

Comment:
Seems like we no longer use `@wordpress/react-i18n` after this PR. Should we remove that dependency and its special handling here: 

https://github.com/Automattic/wp-calypso/blob/a14b6727ffc1ed68b951c95f8a43ca6e1cb15ccc/apps/editing-toolkit/webpack.config.js#L107